plugins/reportal/src/azkaban/reportal/util/Reportal.java

```diff
@@ -46,11 +46,13 @@
 import azkaban.user.Permission;
 import azkaban.user.Permission.Type;
 import azkaban.user.User;
+import azkaban.utils.Props;
 import azkaban.utils.Utils;
 import azkaban.viewer.reportal.ReportalMailCreator;
 import azkaban.viewer.reportal.ReportalTypeManager;
 
 public class Reportal {
+  public static final String REPORTAL_CONFIG_PREFIX = "reportal.config.";
```

Comment:
nit -  you use this prefix in regex the dots will match any characters,  say   reportal.config. will match, reportalAconfigB will match as well .